src/helpers/types/state.ts

```diff
@@ -78,6 +78,7 @@ export interface ICommunitiesState {
     communities: CommunityAttributes[];
     community: CommunityAttributes;
     refreshing: boolean;
+    communityCreationError: any;
```

Comment:
I see it's `null` in some places. Can we improve this and avoid `any`?

Reply:
I followed your API implementation but no problem. I changed from the API call up to states. Fixed.